components/motor/gpiostepper/gpiostepper.go

```diff
@@ -165,6 +169,11 @@ func newGPIOStepper(
 		m.minDelay = time.Duration(mc.StepperDelay * int(time.Microsecond))
 	}
 
+	err = m.enable(ctx, false)
```

Comment:
My goal with this change was to be consistent -- if we are disabling armatures after completing a GoFor command, we should do so on initialization as well. It doesn't make sense [to me] to have them powered on initialization and disabled after running the first command. Here's my proposal:
1. If the user provides enable pins, only enable them when the motor is running
2. If the user wants armatures to always be powered, they should be hard-wired that way.

We don't have an API that would allow users to power armatures on demand.

WDYT?

Reply:
Steppers should always be enabled unless the user disables them on purpose, this is the behaviour that happens when you don't have enable pins. I don't like it disabling after a go for, what if it's holding against a stock piece for another machining operation? That's not safe, relies on the lead screw to hold and unexpected for steppers.

I think maybe in the future we should get rid of the enable pin configuration and if a stepper needs that to see a high/low the user sets them up themselves.

Reply:
I'm more okay with them being disabled at startup and holding after an operation, the inconsistency doesn't bother me as much as the useability problems of disabling the pins.